packages/azure-react/src/generators/app-insights-web/schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/schema",
+    "cli": "nx",
+    "$id": "AppInsightsWeb",
+    "title": "",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "description": "Library name",
+            "$default": {
+                "$source": "argv",
+                "index": 0
+            },
+            "x-prompt": "What name would you like to use for the library?",
+            "pattern": "^[a-zA-Z].*$"
+        },
+        "connectionString": {
+            "type": "string",
+            "description": "The env variable for the connection string.",
+            "x-prompt": "What is the connection string for your app insights instance?"
```

Comment:
Do you mean the connection string itself (e.g. `https://blabla.azure.com` ?) or the env var name? As the description says one and x-prompt says the other.

Reply:
Again, good catch, it is the env variable.